app/src/components/loginPage/LoginPage.tsx

```diff
@@ -1,30 +1,100 @@
 import React, { useCallback, useState, Dispatch, SetStateAction } from "react";
 import { signIn } from "../../firebase/users/UserService";
+import styled from "styled-components";
+// @ts-ignore
+import Elle from "./Elle.svg";
 
 interface Props {
     setLoggedIn: Dispatch<SetStateAction<boolean>>;
 }
 
+const LoginBox = styled.div`
+    min-height: 100vh;
+    padding: 50px 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    max-width: 800px;
+    margin: 0 auto;
+
+    @media only screen and (max-width: 576px) {
+        max-width: 350px;
+    }
+`;
+
+const FormColumn = styled.div`
+    color: var(--purp-selected);
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-top: 20px;
+
+    h1 {
+        font-family: var(--open-sans);
+        font-weight: 800;
+        font-size: 50px;
+    }
+
+    p {
+        font-family: var(--asap);
+        font-weight: normal;
+        font-size: 22px;
+    }
+`;
+
+const ElleImg = styled.img`
+    width: 100%;
+    max-height: calc(100vh - 50px);
+
+    @media only screen and (max-width: 576px) {
+        max-width: 250px;
+    }
+`;
+
 const LoginPage: React.FunctionalComponent<Props> = () => {
     const [loginError, setLoginError] = useState("");
 
     const handleSignIn = useCallback(() => {
-        try {
-            signIn();
-        } catch {
-            setLoginError("There was an error logging in! Please try again.");
-        }
+        (async () => {
+            try {
+                await signIn();
+            } catch {
+                setLoginError(
+                    "There was an error logging in! Please try again."
+                );
+            }
```

Comment:
Can we log the error somewhere please? Even just to the console so we can try to work out what is going on.